packages/fabrix/src/renderer.tsx

```diff
@@ -22,22 +22,24 @@ const decideStrategy = (
   opType: OperationTypeNode,
 ) => {
   const directive = findDirective(directiveNodes);
+  const emptyDirective = {
+    arguments: null,
+  } as const;
+
   switch (opType) {
     case OperationTypeNode.QUERY: {
-      switch (directive?.name) {
-        case "fabrixView":
-          return { type: "view", directive } as const;
-        default:
-          return { type: "view:no-directive" } as const;
-      }
+      return {
+        type: "view",
+        directive:
+          directive?.name === "fabrixView" ? directive : emptyDirective,
+      } as const;
     }
     case OperationTypeNode.MUTATION: {
-      switch (directive?.name) {
-        case "fabrixForm":
-          return { type: "form", directive } as const;
-        default:
-          return { type: "form:no-directive" } as const;
-      }
```

Comment:
I designed these pattern matches, but two cases: `view:no-directive` and `form:no-directive` are actually useless. They are finally dealt with as `view` and `form` in `getFieldConfig` function.